lib/views/reporting/reporting.dart

```diff
@@ -49,138 +69,119 @@ class ReportingViewState extends State<ReportingView> {
         ReportingCardView(
-          question:
-              "Welk diersoort hoort bij de ${widget.interactionType.name.toLowerCase()}?",
+          question: "Diersoort rapportage:",
           step: 1,
           buttonText: "Volgende",
+          goToPreviousPage: _pageController.previousPage,
           onPressed: () {
             _pageController.nextPage(
-              duration: Durations.long1,
+              duration: const Duration(milliseconds: 300),
               curve: Curves.linear,
             );
           },
           onDataChanged: _updateInteractionData,
+          interactionType: widget.interactionType,
         ),
         ReportingCardView(
-          question:
-              "Welk ${_selectedAnimalSpecies != null ? _selectedAnimalSpecies!.substring(0, _selectedAnimalSpecies!.length - 2).toLowerCase() : "diersoort"} hoort bij de ${widget.interactionType.name.toLowerCase()}?",
+          question: "Dier rapportage:",
           step: 2,
           buttonText: "Volgende",
           animalSpecies: _selectedAnimalSpecies,
+          goToPreviousPage: _pageController.previousPage,
           onPressed: () {
             _pageController.nextPage(
-              duration: Durations.long1,
+              duration: const Duration(milliseconds: 300),
               curve: Curves.linear,
             );
           },
           onDataChanged: _updateInteractionData,
+          interactionType: widget.interactionType,
         ),
       ],
-      ReportingCardView(
-        question:
-            "Dit is de locatie van je ${widget.interactionType.name.toLowerCase()}.",
-        step: 3,
-        buttonText: "Volgende",
-        species: _selectedSpecies,
-        onPressed: () {
-          _pageController.nextPage(
-            duration: Durations.long1,
-            curve: Curves.linear,
-          );
-        },
-        onDataChanged: _updateInteractionData,
-      ),
       if (widget.interactionType.id == 2) ...[
         ReportingCardView(
           question: "Heb je nog opmerkingen?",
-          step: 4,
+          step: 3,
           buttonText: "Overslaan",
+          goToPreviousPage: _pageController.previousPage,
           onPressed: () {
             _pageController.nextPage(
-              duration: Durations.long1,
+              duration: const Duration(milliseconds: 300),
               curve: Curves.linear,
             );
           },
           location: _reportLocation,
+          animalSpecies: _selectedAnimalSpecies,
           species: _selectedSpecies,
           interactionType: widget.interactionType,
           onDataChanged: _updateInteractionData,
         ),
       ],
       ReportingCardView(
-        question: "Klopt je rapportage?",
-        step: 5,
+        question: "Overzicht rapportage:",
+        step: 4,
         buttonText: "Rapporteer",
+        goToPreviousPage: _pageController.previousPage,
         onPressed: () {
           _submitReport();
           Navigator.pop(context);
```

Comment:
Je gebruikt hier navigator.pop(context) maar in andere files gebruik je navigator.of(context).pop dit doet allebei hetzelfde dus dat maakt niet veel uit maar het is soms voor het clean houden van code wel chill als je overal hetzelfde gebruikt